src/blurredLocation.js

```diff
@@ -62,6 +61,8 @@ BlurredLocation = function BlurredLocation(options) {
     });
     var geometry = Blurred.responseJSON.results[0].geometry.location;
     options.map.setView([geometry.lat, geometry.lng],options.zoom);
+    $("#lat").val(geometry.lat);
```

Comment:
Let's not do this in here - this method should only geocode. What we do with that info is an interface function. Maybe we do it either on submitting the geocode form input (ui) or when dragging the map (onDrag)?